RCTACPCore.podspec

```diff
@@ -0,0 +1,30 @@
+require "json"
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "RCTACPCore"
+  s.version      = package["version"]
+  s.summary      = "Use Adobe Experience SDKs with React Native"
+  s.author       = "Adobe"
+
+  s.homepage     = "https://github.com/adobe/react-native-acpcore"
+
+  s.license      = "Apache License"
+  s.platform     = :ios
+
+  s.source       = { :git => "https://github.com/adobe/react-native-acpcore.git", :tag => "#{s.version}" }
+
+  s.source_files  = "ios/**/*.{h,m}"
+  s.requires_arc = true
+
+  s.dependency "React"  
+  s.dependency "ACPGriffonBeta", "~> 0.0"
```

Comment:
I don't believe all these dependencies are required for all use cases. It should probably only depend on React IMO.

Reply:
Thanks! True. Updated according, let me know if ok. - Do we need the frameworks and libraries?

Reply:
I believe we need to link all the libraries inside https://github.com/adobe/react-native-acpcore/tree/master/ios/libs